src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```diff
@@ -1,12 +1,16 @@
 import Box from '@mui/material/Box'
-
+import CategoryDropdown from '~/containers/category-dropdown/CategoryDropdown'
 import { styles } from '~/containers/tutor-home-page/subjects-step/SubjectsStep.styles'
+import img from '~/assets/img/tutor-home-page/become-tutor/study-category.svg'
 
 const SubjectsStep = ({ btnsBox }) => {
   return (
     <Box sx={styles.container}>
+      <Box sx={styles.imgContainer}>
+        <Box component='img' src={img} sx={styles.img} />
+      </Box>
       <Box sx={styles.rigthBox}>
-        Subjects step
+        <CategoryDropdown />
```

Comment:
I don't think this is a correct component to use here. At least not like this. Fro example, placeholder in the task should be Main tutoring category, not Category. Plus, in this case users should be able to create new categories. This component was created for different use, so you should create another dropdown or try adapt this one, if it is possible

Reply:
CategoryDropdown is adapted now. Please, check last code updates.